CHANGELOG.md

```diff
@@ -4,43 +4,16 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 2.17.0 (In Progress)
 ====================
-**Preflight API Pilot**
+**New Features**
 -----------------------
 
-- This release adds a new experimental API `runPreflight` to help test the connectivity with twilio servers.
-The API connects two Peer connections using Twilio's turn servers. It publishes synthetic audio and video
-tracks from one participant and ensures that other participant receives media on those tracks.
-After successfully verifying the connectivity, it generates a report about statistics of the connection.
+This release includes the **Preflight API Public Beta** (`runPreflight`) to help test connectivity with Twilio servers and estimate media quality for a Participant. It can be used to detect issues prior to joining a Video Room or as part of a troubleshooting page.  
 
-**Few things to note:**
-- This function uses <a href="https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API"> web  audio API's.</a>
-Browser's  autoplay policies sometimes require user action before accessing these APIs. Please ensure that this API
-is called in response to user action like a button click.
-- `preflightTest` emits `failed` event to indicate test failures. Please refer to
-[this guide](https://www.twilio.com/docs/video/build-js-video-application-recommendations-and-best-practices#connection-errors)
-for interpreting common errors.
+The API connects two peer connections using Twilio's TURN servers. It publishes synthetic audio and video tracks from one participant and ensures that other participant receives media on those tracks. After successfully verifying connectivity, it generates a report with statistics on the connection.
```

Comment:
Is it a peer connection to itself or two separate peer connections to itself? Seems like it is a single peer connection to itself. 
You should also mention that it connects to Twilio signaling servers.

Reply:
its two peer connections. +1 on mentioning Twilio signaling and turn servers

Reply:
ok, what way do we use the peer connections? Are we sending video/audio on both?

Reply:
we have to use two peerConnections to establish connectivity - one for Alice another for Bob - Alice's PC publishes media, Bob's PC subscribes to it.